11:28:00,589 (main) ERROR [DataNucleus.SchemaTool] - Error creating NucleusContext
No se puede leer el archivo de propiedades "datanucleus.properties"
org.datanucleus.exceptions.NucleusUserException: No se puede leer el archivo de propiedades "datanucleus.properties"
	at org.datanucleus.Configuration.getPropertiesFromPropertiesFile(Configuration.java:693)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:294)
Caused by: java.lang.NullPointerException: inStream parameter is null
	at java.base/java.util.Objects.requireNonNull(Objects.java:233)
	at java.base/java.util.Properties.load(Properties.java:407)
	at org.datanucleus.Configuration.getPropertiesFromPropertiesFile(Configuration.java:687)
	... 1 more
Nested Throwables StackTrace:
java.lang.NullPointerException: inStream parameter is null
	at java.base/java.util.Objects.requireNonNull(Objects.java:233)
	at java.base/java.util.Properties.load(Properties.java:407)
	at org.datanucleus.Configuration.getPropertiesFromPropertiesFile(Configuration.java:687)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:294)
11:35:21,054 (main) ERROR [DataNucleus.SchemaTool] - Error creating NucleusContext
No se puede leer el archivo de propiedades "datanucleus.properties"
org.datanucleus.exceptions.NucleusUserException: No se puede leer el archivo de propiedades "datanucleus.properties"
	at org.datanucleus.Configuration.getPropertiesFromPropertiesFile(Configuration.java:693)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:294)
Caused by: java.lang.NullPointerException: inStream parameter is null
	at java.base/java.util.Objects.requireNonNull(Objects.java:233)
	at java.base/java.util.Properties.load(Properties.java:407)
	at org.datanucleus.Configuration.getPropertiesFromPropertiesFile(Configuration.java:687)
	... 1 more
Nested Throwables StackTrace:
java.lang.NullPointerException: inStream parameter is null
	at java.base/java.util.Objects.requireNonNull(Objects.java:233)
	at java.base/java.util.Properties.load(Properties.java:407)
	at org.datanucleus.Configuration.getPropertiesFromPropertiesFile(Configuration.java:687)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:294)
11:36:37,542 (main) ERROR [DataNucleus.SchemaTool] - Error creating NucleusContext
No se puede leer el archivo de propiedades "datanucleus.properties"
org.datanucleus.exceptions.NucleusUserException: No se puede leer el archivo de propiedades "datanucleus.properties"
	at org.datanucleus.Configuration.getPropertiesFromPropertiesFile(Configuration.java:693)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:294)
Caused by: java.lang.NullPointerException: inStream parameter is null
	at java.base/java.util.Objects.requireNonNull(Objects.java:233)
	at java.base/java.util.Properties.load(Properties.java:407)
	at org.datanucleus.Configuration.getPropertiesFromPropertiesFile(Configuration.java:687)
	... 1 more
Nested Throwables StackTrace:
java.lang.NullPointerException: inStream parameter is null
	at java.base/java.util.Objects.requireNonNull(Objects.java:233)
	at java.base/java.util.Properties.load(Properties.java:407)
	at org.datanucleus.Configuration.getPropertiesFromPropertiesFile(Configuration.java:687)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:294)
16:24:46,849 (main) ERROR [DataNucleus.Datastore] - Un error estaba tirado con DROP TABLE `RETO` CASCADE : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

16:24:46,851 (main) ERROR [DataNucleus.Datastore.Schema] - Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

16:24:46,853 (main) ERROR [DataNucleus.SchemaTool] - DataNucleus SchemaTool ha lanzado una excepción. Por favor consulte el Log para ver más detalles: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

org.datanucleus.exceptions.NucleusUserException: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:219)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
Caused by: java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	... 4 more
Nested Throwables StackTrace:
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
16:25:09,477 (main) ERROR [DataNucleus.Datastore] - Un error estaba tirado con DROP TABLE `RETO` CASCADE : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

16:25:09,478 (main) ERROR [DataNucleus.Datastore.Schema] - Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

16:25:09,481 (main) ERROR [DataNucleus.SchemaTool] - DataNucleus SchemaTool ha lanzado una excepción. Por favor consulte el Log para ver más detalles: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

org.datanucleus.exceptions.NucleusUserException: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:219)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
Caused by: java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	... 4 more
Nested Throwables StackTrace:
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
19:30:59,987 (main) ERROR [DataNucleus.Datastore] - Un error estaba tirado con DROP TABLE `RETO` CASCADE : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

19:30:59,988 (main) ERROR [DataNucleus.Datastore.Schema] - Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

19:30:59,991 (main) ERROR [DataNucleus.SchemaTool] - DataNucleus SchemaTool ha lanzado una excepción. Por favor consulte el Log para ver más detalles: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

org.datanucleus.exceptions.NucleusUserException: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:219)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
Caused by: java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	... 4 more
Nested Throwables StackTrace:
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
19:31:47,742 (main) ERROR [DataNucleus.Datastore] - Un error estaba tirado con DROP TABLE `RETO` CASCADE : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

19:31:47,746 (main) ERROR [DataNucleus.Datastore.Schema] - Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

19:31:47,754 (main) ERROR [DataNucleus.SchemaTool] - DataNucleus SchemaTool ha lanzado una excepción. Por favor consulte el Log para ver más detalles: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

org.datanucleus.exceptions.NucleusUserException: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:219)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
Caused by: java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	... 4 more
Nested Throwables StackTrace:
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
19:32:24,766 (main) ERROR [DataNucleus.Datastore] - Un error estaba tirado con DROP TABLE `RETO` CASCADE : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

19:32:24,769 (main) ERROR [DataNucleus.Datastore.Schema] - Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

19:32:24,775 (main) ERROR [DataNucleus.SchemaTool] - DataNucleus SchemaTool ha lanzado una excepción. Por favor consulte el Log para ver más detalles: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

org.datanucleus.exceptions.NucleusUserException: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:219)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
Caused by: java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	... 4 more
Nested Throwables StackTrace:
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
19:32:53,165 (main) ERROR [DataNucleus.Datastore] - Un error estaba tirado con DROP TABLE `RETO` CASCADE : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

19:32:53,167 (main) ERROR [DataNucleus.Datastore.Schema] - Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

19:32:53,171 (main) ERROR [DataNucleus.SchemaTool] - DataNucleus SchemaTool ha lanzado una excepción. Por favor consulte el Log para ver más detalles: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

org.datanucleus.exceptions.NucleusUserException: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:219)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
Caused by: java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	... 4 more
Nested Throwables StackTrace:
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
19:33:52,307 (main) ERROR [DataNucleus.Datastore] - Un error estaba tirado con DROP TABLE `RETO` CASCADE : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

19:33:52,308 (main) ERROR [DataNucleus.Datastore.Schema] - Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

19:33:52,311 (main) ERROR [DataNucleus.SchemaTool] - DataNucleus SchemaTool ha lanzado una excepción. Por favor consulte el Log para ver más detalles: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

org.datanucleus.exceptions.NucleusUserException: Una excepcion estaba tirada durante la borracion de algunas class(es) : Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)

	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:219)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
Caused by: java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	... 4 more
Nested Throwables StackTrace:
java.sql.SQLException: Cannot drop table 'reto' referenced by a foreign key constraint 'USUARIO_RETOS_FK2' on table 'usuario_retos'.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:764)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.datasource.dbcp2.DelegatingStatement.execute(DelegatingStatement.java:194)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatement(AbstractTable.java:799)
	at org.datanucleus.store.rdbms.table.AbstractTable.executeDdlStatementList(AbstractTable.java:749)
	at org.datanucleus.store.rdbms.table.AbstractTable.drop(AbstractTable.java:507)
	at org.datanucleus.store.rdbms.DeleteTablesSchemaTransaction.run(DeleteTablesSchemaTransaction.java:210)
	at org.datanucleus.store.rdbms.AbstractSchemaTransaction.execute(AbstractSchemaTransaction.java:118)
	at org.datanucleus.store.rdbms.RDBMSStoreManager.deleteSchemaForClasses(RDBMSStoreManager.java:4202)
	at org.datanucleus.store.schema.SchemaTool.deleteSchemaForClasses(SchemaTool.java:512)
	at org.datanucleus.store.schema.SchemaTool.main(SchemaTool.java:427)
